sdk/storage/storage-file-datalake/src/utils/utils.common.ts

```diff
@@ -537,29 +537,30 @@ export function iEqual(str1: string, str2: string): boolean {
   return str1.toLocaleLowerCase() === str2.toLocaleLowerCase();
 }
 
+/**
+ * Extracts account name from the blobEndpointUrl
+ * @param {string} blobEndpointUrl blobEndpointUrl to extract the account name from
+ * @returns {string} account name
+ */
 export function getAccountNameFromUrl(blobEndpointUrl: string): string {
```

Comment:
Hey @XiaoningLiu,
I had updated only this one instance of `getAccountNameFromUrl` in the `storage-file-datalake` related to the name properties, please verify if there are more instances to take care of.

Reply:
Sounds good.